packages/web/src/store/selectors/scanDisplayOptions/scanDisplayOptions.ts

```diff
@@ -46,14 +47,29 @@ export const makeSelectSortedAndFilteredScanPackages = () =>
   createSelector(
     [makeSelectScanPackagesByUrl(), makeSelectOptimizedScanDisplayOptions()],
     (packages = [], displayOptions) => {
-      const { packageFilters, packageSorters } = displayOptions;
+      const { searchText, packageFilters, packageSorters } = displayOptions;
 
-      const filteredPackages = packages.filter((pkg) =>
+      const searchValue = searchText.toLowerCase();
+
+      // TODO: looks excessive with Array.isArray, but eslint will get trigger for 'use nullish coalescing' otherwise
+      //       with just ||, probably could be simplified
```

Comment:
Probably you could suppress the errors by adding `!!` to each part of OR-clause, so then you won't need that check